README.md

```diff
@@ -1,20 +1,105 @@
-# OpenApocalypse
-Linux: [![Linux Build Status](https://travis-ci.org/OpenApoc/OpenApoc.svg?branch=master)](https://travis-ci.org/OpenApoc/OpenApoc)
-Windows: [![Windows Build Status](https://ci.appveyor.com/api/projects/status/61d093s2rl5q55bm/branch/master?svg=true
-)](https://ci.appveyor.com/project/JonnyH/openapoc-iq0ey/)
-
-The OpenApocalypse project aims to build a open-source
-clone of the original X-COM: Apocalypse strategy game.
-
-OpenApoc is in early development - it is not yet 'playable' as a game.
-All help is appreciated, but there's little point trying to 'play' it as of yet.
-
-[See our Trello TO-DO list for more info.](https://trello.com/b/lX5Y3DwR/openapoc)
-
-## Requirements
-
-* This assumes that you have the file 'cd.iso' - a copy of the original X-Com Apocalypse CD (This can be acquired from [Steam](http://store.steampowered.com/app/7660/) - this is _required_ to run)
-  * If you have the real cd, you can copy all the included files to a directory called "cd.iso"
+# OpenApocalypse [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=Are%20you%20fan%20X-Com%20Apocalypse?%20OpenApoc%20its%20clone%20of%20this%20great%20game%20-%20contribute!%20https://github.com/OpenApoc/OpenApoc&hashtags=games,openapoc,xcom)
+> OpenApoc is an open-source clone of the original [X-COM: Apocalypse](http://www.ufopaedia.org/index.php/Apocalypse), licensed under the MIT and written in C++ / SDL2. It was originally founded by PmProg in July 2014, and has since grown in to [community](http://www.ufopaedia.org/index.php/Credits_(OpenApoc)). 
```

Comment:
It's not really a "clone" - it's a re-implementation of the game engine.

A "Clone" means it's a standalone game, based somewhat on the original, while openapoc aims to be *exactly* the same game, and requires the original's resource files to run.